mesa/space.py

```diff
@@ -34,12 +37,44 @@ class Grid(object):
         get_cell_list_contents: Returns the contents of a list of cells ((x,y) tuples)
     '''
 
-    width = None
-    height = None
-    torus = False
-    grid = None
     default_val = lambda s: None
 
+
+    class GridOccupiedIter:
```

Comment:
@gcallah - this is a much nicer way of doing this. 
`GridOccupiedIter` is not intuitive for non-programmers to use. Without understanding Python, you might not know what that is. Suggestions for another names? 
@dmasad - do you have any thoughts on this?